daml-assistant/daml-sdk/validate.sh

```diff
@@ -31,3 +31,8 @@ done
 for cmd in sandbox sandbox-classic; do
     $JAVA -jar $SDK_EE $cmd --help | grep -q profile-dir
 done
+
+$JAVA -jar $SDK_EE trigger-service --help | grep -q oracle
```

Comment:
I'd find this easier to read as
```suggestion
if ! ($JAVA -jar $SDK_EE trigger-service --help | grep -q oracle); then
  exit 1
fi
```